kentik_api_library/tests/unit/synthetics/test_traces.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime, timezone
+
+from google.protobuf.timestamp_pb2 import Timestamp
+
+import kentik_api.generated.kentik.synthetics.v202202.synthetics_pb2 as pb
+from kentik_api.public.types import ID, IP
+from kentik_api.synthetics.synth_client import KentikSynthClient
+from kentik_api.synthetics.synth_tests.traces import Location, NetNode, Path, PathTrace, Stats, TraceHop, TraceResponse
+from tests.unit.synthetics.stub_api_connector import StubAPISyntheticsConnector
+
+PB_TRACES = pb.GetTraceForTestResponse(
+    nodes={
+        "10.100.20.200": pb.NetNode(
+            ip="10.100.20.200",
+            asn=1,
+            as_name="as_name",
+            location=pb.Location(latitude=18.0, longitude=54.0, country="Poland", region="Pomeranian", city="Gdańsk"),
+            dns_name="aws.route53.com",
+            device_id="50",
+            site_id="100",
+        )
+    },
+    paths=[
+        pb.Path(
+            agent_id="1000",
+            target_ip="10.100.20.200",
+            hop_count=pb.Stats(average=50, min=10, max=90),
+            max_as_path_length=11,
```

Comment:
It does not matter much, but the test data are inconsistent. If `max_as_path_length` is 11, then there needs to be a trace with at least 11 hops. It surely would be useful to have a response with at least 2 hops and more than 1 trace.

Reply:
Updating